site/ja/probability/examples/Gaussian_Process_Latent_Variable_Model.ipynb

```diff
@@ -85,7 +85,7 @@
       "source": [
         "## GP を適用する: 回帰と潜在変数モデル\n",
         "\n",
-        "GP を使用する 1 つの方法として、回帰が挙げられます。入力 ${x_i}*{i=1}^N$（インデックスセットの要素）と観測値 ${y_i}*{i=1}^N$ の形式の一連の観測データが与えられた場合、これらを使用して、新しいポイントセット ${x_j^*}_{j=1}^M$ で事後予測分布を形成できます。分布はすべてガウスであるため、これは簡単な線形代数に要約されます（ただし、必要な計算では、データポイントの数にランタイム *cubeic* があり、データポイントの数に 2 次の空間が必要です。これは GP の使用における主要な制限要因であり、現在の多くの研究は、正確な事後推論の計算上実行可能な代替案に焦点を合わせています。GP 回帰の詳細な説明については、[TFP における GP 回帰の Colab](https://colab.research.google.com/github/tensorflow/probability/blob/main/tensorflow_probability/examples/jupyter_notebooks/Gaussian_Process_Regression_In_TFP.ipynb) をご覧ください。\n",
+        "GP を使用する 1 つの方法として、回帰が挙げられます。入力 ${x_i}{em0}{i=1}^N$（インデックスセットの要素）と観測値 ${y_i}{/em0}{i=1}^N$ の形式の一連の観測データが与えられた場合、これらを使用して、新しいポイントセット ${x_j^*}_{j=1}^M$ で事後予測分布を形成できます。分布はすべてガウスであるため、これは簡単な線形代数に要約されます（ただし、必要な計算では、データポイントの数にランタイム <em>cubeic</em> があり、データポイントの数に 2 次の空間が必要です。これは GP の使用における主要な制限要因であり、現在の多くの研究は、正確な事後推論の計算上実行可能な代替案に焦点を合わせています。GP 回帰の詳細な説明については、<a>TFP における GP 回帰の Colab</a> をご覧ください。\n",
```

Comment:
"cubeic" is not a word in English. I assume this is supposed to be "cubic." What happened here? It seems like the word is spelled correctly in the original: https://www.tensorflow.org/probability/examples/Gaussian_Process_Latent_Variable_Model